core/src/main/java/org/eclipse/hono/util/NotificationDeviceConnectionConstants.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 1.0 which is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ */
+
+package org.eclipse.hono.util;
+
+import io.vertx.core.json.JsonObject;
+
+/**
+ * Constants used for command readiness notifications of a device.
+ */
+public final class NotificationDeviceConnectionConstants extends NotificationConstants {
```

Comment:
this doesn't seem to have anything to do with a device indicating readiness to receive a command, does it?

Reply:
Was meant as an integration with the `connected/disconnected` event - will eliminate it for clarity.